virtcontainers/clh_test.go

```diff
@@ -172,7 +172,14 @@ func TestCloudHypervisorBootVM(t *testing.T) {
 }
 
 func TestCloudHypervisorCleanupVM(t *testing.T) {
-	clh := &cloudHypervisor{}
+	store, err := persist.GetDriver()
+	if err != nil {
+		t.Errorf("persist.GetDriver() unexpected error != %v", err)
```

Comment:
Nit: The assert syntax save you two lines of code :smile:

```go
assert.NoError(err)
```

Reply:
Fixed, thanks